config/test.exs

```diff
@@ -13,6 +13,9 @@ config :hex_web, HexWeb.Endpoint,
   http: [port: 4001],
   server: false
 
+config :hex_web, HexWeb.Mailer,
+  adapter: Bamboo.LocalAdapter
```

Comment:
I think we need to add it to `dev.exs` and `hex.exs` as well. (`hex.exs` is used by hex client when running it's integration tests).

Previously, we were choosing adapter (SES vs Local) based on the presence of an env variable in `config.exs`, so I think we can go back to that, but keeping it in separate files is ok too.